src/main/java/no/nav/pto/veilarbportefolje/oppfolging/OppfolgingStartetService.java

```diff
@@ -29,11 +30,14 @@
     @Override
     public void behandleKafkaMeldingLogikk(OppfolgingStartetDTO dto) {
         mapAktoerTilPersonId(dto.getAktorId());
+        startOppfolging(dto.getAktorId(), dto.getOppfolgingStartet());
+    }
```

Comment:
is this method still in use? behandleKafkaMeldingLogikk

Reply:
I guess that if this one can be removed, then we can also remove the one in oppfølging avsluttet

Reply:
Yep, agree...will be fixed